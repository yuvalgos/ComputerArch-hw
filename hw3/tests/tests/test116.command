./dflow_calc tests/test116.dat tests/test116.in p82 p167 p184 d190 d84 p74 p25 d144 p123 p190 d181 d90 d3 p56 p41 d113 p128 d99 d75 d67 d101 d148 p17 p63 p142 p119 d196 d17 p179 p120 p178 d146 d12 d158 p102 p43 p13 p160 p131 d69 p35 d114 d198 d89 p6 p119 p2 p5 d171 p24 d184 d148 p162 d157 d135 p71 d174 d114 p116 d31 p143 d16 d57 p56 p55 p135 p3 d175 p38 p93 d133 d95 d77 d85 d188 d114 d53 d176 p7 p176 d129 p117 p56 d189 d99 p187 d110 p182 p104 d13 d172 p119 p145 d114 d191 p194 d20 p9 p161 p196 d35 d38 p99 d120 d27 p117 p69 d61 p147 p121 d160 p170 p8 p153 d184 p142 p80 p180 d19 d93 p18 d133 d144 d198