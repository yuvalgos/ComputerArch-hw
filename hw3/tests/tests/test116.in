7 10 12 4
6 12 31 12
5 13 21 16
8 24 27 17
0 5 24 21
1 25 18 0
2 2 4 27
0 28 24 26
1 28 6 12
1 22 14 21
7 27 26 11
5 12 5 24
7 3 8 10
3 13 11 9
4 18 9 22
7 5 30 15
8 31 26 8
3 24 15 19
6 23 18 15
1 21 2 4
0 11 30 0
1 30 1 12
6 14 17 8
0 6 3 24
6 25 23 11
2 13 16 6
4 24 30 18
7 27 0 2
8 4 10 25
0 7 1 0
0 15 1 24
8 19 16 6
5 17 20 13
5 31 18 31
3 25 4 5
5 20 13 18
0 3 26 22
1 0 15 5
6 5 27 30
5 6 15 5
0 16 6 1
0 4 23 0
1 7 27 7
2 4 19 28
6 24 11 10
2 4 14 8
5 5 10 17
5 1 19 3
1 20 16 13
5 2 2 7
8 10 14 4
6 15 5 6
3 19 4 27
0 15 15 13
3 13 24 9
7 30 11 28
3 6 25 1
5 21 31 28
6 4 19 14
2 10 25 5
1 28 23 3
8 0 1 0
3 7 30 1
7 13 0 4
3 27 1 13
3 14 0 9
1 24 4 16
7 20 29 22
5 3 25 30
7 9 17 18
2 2 7 4
4 30 14 31
1 12 17 22
1 23 28 17
0 26 6 5
0 14 14 29
7 7 2 7
4 23 12 23
4 10 6 9
3 24 21 15
1 5 26 8
7 1 2 4
0 8 2 17
3 16 17 1
7 20 1 28
0 6 17 9
0 4 21 31
0 24 6 16
2 29 28 8
6 3 20 1
1 6 15 17
2 27 21 11
1 24 30 24
5 9 3 2
4 2 12 9
1 25 22 30
1 16 15 7
2 11 25 25
5 4 24 20
6 8 0 18
3 22 3 29
8 5 11 2
0 5 13 15
3 11 31 21
8 13 31 1
7 22 6 30
3 6 5 8
6 23 11 25
7 3 28 3
2 31 30 19
3 31 7 8
0 29 7 4
6 19 0 16
3 15 24 5
0 27 30 0
8 14 12 5
0 28 0 22
3 16 19 29
5 21 2 17
3 30 31 26
1 6 3 0
8 26 24 15
3 1 31 27
0 22 9 2
5 3 23 14
4 22 14 20
3 14 2 7
5 14 26 30
0 14 6 22
5 17 26 21
7 14 0 17
6 25 19 12
8 5 16 4
4 7 11 12
0 20 13 10
8 11 19 10
5 26 24 12
7 4 16 1
3 30 4 29
1 26 15 29
5 1 15 11
4 29 22 24
7 14 29 17
3 25 31 8
3 0 26 10
5 24 23 1
5 27 23 1
3 27 9 13
6 4 11 30
8 10 26 18
3 23 7 6
7 1 0 12
2 15 4 16
4 26 27 3
8 7 16 25
5 10 13 10
3 31 17 9
4 21 18 14
0 22 0 10
7 30 19 27
0 22 23 15
7 17 0 28
6 24 25 14
4 0 0 8
0 20 6 9
5 26 18 30
1 13 19 28
4 26 3 29
7 13 15 19
5 21 6 15
6 2 0 30
2 24 16 26
4 19 11 15
0 24 13 17
6 3 27 26
7 14 24 10
6 1 6 2
3 29 18 23
7 0 29 9
1 19 8 8
2 23 29 31
7 25 3 19
6 8 31 19
2 12 30 30
1 24 15 2
4 29 14 14
1 10 16 27
3 15 0 25
7 18 21 12
2 19 26 11
1 1 18 2
1 4 31 27
7 10 1 8
7 10 31 24
3 3 1 13
4 11 23 23
5 15 20 14
2 23 20 31
8 9 1 4
4 3 3 27
