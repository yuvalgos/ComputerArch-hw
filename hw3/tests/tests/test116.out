Reading the opcodes latency info from tests/test116.dat ... Got latency for 9 opcodes
Reading the program file tests/test116.in ... Found 200 instructions
getProgDepth()==254
getDepDepth(82)==88
getDepDepth(167)==165
getDepDepth(184)==199
getInstDeps(190)=={188,170}
getInstDeps(84)=={81,60}
getDepDepth(74)==85
getDepDepth(25)==40
getInstDeps(144)=={139,78}
getDepDepth(123)==167
getDepDepth(190)==228
getInstDeps(181)=={174,179}
getInstDeps(90)=={53,32}
getInstDeps(3)=={-1,-1}
getDepDepth(56)==78
getDepDepth(41)==57
getInstDeps(113)=={92,102}
getDepDepth(128)==165
getInstDeps(99)=={61,14}
getInstDeps(75)=={65,-1}
getInstDeps(67)=={-1,9}
getInstDeps(101)=={97,94}
getInstDeps(148)=={135,138}
getDepDepth(17)==0
getDepDepth(63)==99
getDepDepth(142)==193
getDepDepth(119)==123
getInstDeps(196)=={164,175}
getInstDeps(17)=={-1,-1}
getDepDepth(179)==116
getDepDepth(120)==162
getDepDepth(178)==179
getInstDeps(146)=={107,140}
getInstDeps(12)=={-1,0}
getInstDeps(158)=={144,155}
getDepDepth(102)==118
getDepDepth(43)==63
getDepDepth(13)==0
getDepDepth(160)==196
getDepDepth(131)==127
getInstDeps(69)=={32,14}
getDepDepth(35)==45
getInstDeps(114)=={71,61}
getInstDeps(198)=={190,191}
getInstDeps(89)=={84,81}
getDepDepth(6)==0
getDepDepth(119)==123
getDepDepth(2)==0
getDepDepth(5)==0
getInstDeps(171)=={117,167}
getDepDepth(24)==31
getInstDeps(184)=={152,170}
getInstDeps(148)=={135,138}
getDepDepth(162)==212
getInstDeps(157)=={14,142}
getInstDeps(135)=={112,78}
getDepDepth(71)==106
getInstDeps(174)=={147,167}
getInstDeps(114)=={71,61}
getDepDepth(116)==136
getInstDeps(31)=={-1,23}
getDepDepth(143)==123
getInstDeps(16)=={-1,-1}
getInstDeps(57)=={33,8}
getDepDepth(56)==78
getDepDepth(55)==20
getDepDepth(135)==127
getDepDepth(3)==0
getInstDeps(175)=={173,155}
getDepDepth(38)==41
getDepDepth(93)==143
getInstDeps(133)=={103,72}
getInstDeps(95)=={9,71}
getInstDeps(77)=={72,73}
getInstDeps(85)=={32,69}
getInstDeps(188)=={169,183}
getInstDeps(114)=={71,61}
getInstDeps(53)=={51,25}
getInstDeps(176)=={120,170}
getDepDepth(7)==23
getDepDepth(176)==199
getInstDeps(129)=={121,118}
getDepDepth(117)==141
getDepDepth(56)==78
getInstDeps(189)=={167,135}
getInstDeps(99)=={61,14}
getDepDepth(187)==212
getInstDeps(110)=={76,99}
getDepDepth(182)==200
getDepDepth(104)==107
getInstDeps(13)=={-1,-1}
getInstDeps(172)=={135,152}
getDepDepth(119)==123
getDepDepth(145)==163
getInstDeps(114)=={71,61}
getInstDeps(191)=={156,147}
getDepDepth(194)==231
getInstDeps(20)=={-1,-1}
getDepDepth(9)==0
getDepDepth(161)==151
getDepDepth(196)==236
getInstDeps(35)=={25,14}
getInstDeps(38)=={27,21}
getDepDepth(99)==99
getInstDeps(120)=={108,61}
getInstDeps(27)=={-1,6}
getDepDepth(117)==141
getDepDepth(69)==58
getInstDeps(61)=={47,37}
getDepDepth(147)==156
getDepDepth(121)==127
getInstDeps(160)=={150,152}
getDepDepth(170)==182
getDepDepth(8)==17
getDepDepth(153)==163
getInstDeps(184)=={152,170}
getDepDepth(142)==193
getDepDepth(80)==87
getDepDepth(180)==200
getInstDeps(19)=={6,-1}
getInstDeps(93)=={89,70}
getDepDepth(18)==14
getInstDeps(133)=={103,72}
getInstDeps(144)=={139,78}
getInstDeps(198)=={190,191}
